.github/ISSUE_TEMPLATE/feedback.yml

```
@@ -0,0 +1,44 @@
+name: Feedback on Django OTP WebAuthn documentation
+description: Provide feedback on the Django OTP WebAuthn documentation.
+title: "[Feedback]: "
```

@activus-d shall we assign labels as well?

Let's add a label to these issues as well.

```suggestion
title: "[Feedback]: "
labels: ["Documentation"]
```